client/ayon_motionbuilder/api/testing/tests.py

```diff
@@ -0,0 +1,74 @@
+import contextlib
+import pyblish.util
+from pyfbsdk import FBSystem
+from ayon_core.pipeline import (
+    registered_host
+)
+from ayon_core.pipeline.create import CreateContext
+
+from . import lib
+
+# Needed for transition phase for asset/subset renaming. Can be hardcoded once
+# transition is done.
+product_key_name = "productName"
+product_type_key_name = "productType"
+
+
+def test_create():
+    """Test re-creating instances in workfile.
+    """
+
+    host = registered_host()
+    context = CreateContext(host)
+    # TODO: replace the hardcoded of the data
+    data = {
+        "creator_identifier": "io.ayon.creators.motionbuilder.animation",
+        "creator_attributes": {
+            "EmbedMedia": True,
+            "SaveSelectedModelsOnly": True,
+            "KeepTransformHierarchy": True
+        }
```

Comment:
Would be great to know any way to get it through the context.

Reply:
If you'd like to get those through the context - how would you then like to 'choose' the creator if not by the identifier? The class name? or what did you have in mind?


Also, it may be good to rename this test to `test_create_animation()` since it's specific to that creator.

Reply:
> If you'd like to get those through the context - how would you then like to 'choose' the creator if not by the identifier? The class name? or what did you have in mind?
> 
> Also, it may be good to rename this test to `test_create_animation()` since it's specific to that creator.

For this host, it is okay with the data being hardcoded as it is currently just having one creator for animation (and one autocreator for workfile). I am still finding the way for storing the data when there is more than one product type